client/my-sites/site-settings/related-posts/related-content-preview.jsx

```diff
@@ -11,11 +11,14 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import FormLabel from 'components/forms/form-label';
+import imgCatBlog from 'gutenberg/extensions/related-posts/cat-blog.png';
```

Comment:
The single fact that we have to import images in site settings from our Gutenberg extensions is enough reason to NOT move those images there, but keep them where they were before. Not to mention that we might be using these somewhere else, so I'd refrain from moving them.